redwood-lazylayout-view/src/main/kotlin/app/cash/redwood/lazylayout/view/ViewLazyList.kt

```diff
@@ -123,6 +122,20 @@ internal open class ViewLazyListImpl(
     recyclerView.adapter = adapter
   }
 
+  override fun width(width: Constraint) {
+    recyclerView.updateLayoutParams {
+      this.width = if (width == Constraint.Fill) MATCH_PARENT else WRAP_CONTENT
+    }
+    recyclerView.requestLayout()
```

Comment:
Setting layout parameters will automatically request layout. Although it doesn't really hurt to have it here...

Reply:
Oh, true! I'll remove it cause I don't want to imply that the code is doing more than it is.